src/server/pps/server/api_server.go

```diff
@@ -918,6 +918,9 @@ func (a *apiServer) StartPod(ctx context.Context, request *ppsserver.StartPodReq
 			OutputCommit: commit,
 		},
 	})
+	if err != nil {
+		return nil, err
+	}
```

Comment:
This was actually a latent bug that I encountered working on this and decided to fix. We weren't actually checking the error that gets returned above.